sherpa/astro/datastack/tests/test_datastack.py

```diff
@@ -680,7 +680,7 @@ def test_case_7(self):
 
         assert f == [1, 2]
 
-        f = datastack.query_by_obsid('7867')
+        f = datastack.query_by_obsid(7867)
```

Comment:
The current doc string for `query_by_obsid` startes that the value can be a str or an integer. Does the str option no-longer work (and if so, how much work would it be to restore)?